prac_06/guitars.py

```diff
@@ -0,0 +1,31 @@
+"""
+guitars.py
+Estimate - 1 hour
+Actual - 45 minutes
+"""
+
+from prac_06.guitar import Guitar
+
+
+def main():
+    guitars = []
+    print("My guitars!")
+    name = input("Name: ")
+    while name != "":
+        year = int(input("Year: "))
+        cost = float(input("Cost: $"))
+        guitar = Guitar(name, year, cost)
+        guitars.append(guitar)
+        print(f"{guitar} added.")
+        name = input("Name: ")
+
+    # guitars.append(Guitar("Gibson L-5 CES", 1922, 16035.40))
```

Comment:
A comment explaining that these are for testing could be appropriate

Reply:
thats right
thanks.